### ISSUE TITLE
Text-Align for texts of different lengths

### ISSUE THREAD

**Issue by teeban10** [Hopding/pdf-lib#152]
So I was working on a project which will embed names on the white box and those names differs in terms of length. So I was wondering a method to make the text to appear on the center of the field no matter how long the names are. 
P/S : thanks for making this great library been a great help for me

**Comment by teeban10** [comment]
Okay I solved the issue it it was based on issue #10  just follow the method 4 and adjust the height according to your requirements. In my case I did this 
```
firstPage.drawText(name , {
        x: (width - textWidth) / 2,
        y: (height-368) - (textSize * 3),
        size: textSize,
        font: contName,
        color: rgb(0, 0, 0),
        rotate: degrees(0),
    });
```

**Comment by teeban10** [comment]
Okay I solved the issue it it was based on issue #10 just follow the method 4 and adjust the height according to your requirements. In my case I did this

firstPage.drawText(name , {
        x: (width - textWidth) / 2,
        y: (height-368) - (textSize * 3),
        size: textSize,
        font: contName,
        color: rgb(0, 0, 0),
        rotate: degrees(0),
    });
